linkerd/examples/thriftmux.yaml

```diff
@@ -0,0 +1,21 @@
+# A simplistic thriftmux-routing linkerd configuration.
+
+# An example service discovery that backend uses the filesystem to resolve endpoints.
+namers:
+- kind: io.l5d.fs
+  prefix: /thriftmux
+  rootDir: linkerd/examples/io.l5d.fs
+
+# A simple thriftmux router that looks up each thriftmux method in io.l5d.fs.
+routers:
+- protocol: thriftmux
```

Comment:
I don't think this example routes correctly, but it looks like neither does the thrift.yaml example.  Perhaps we can just fix both of them in a followup review.